panther_analysis_tool/schemas.py

```diff
@@ -29,9 +29,10 @@
     r"\.Instance|Redshift\.Cluster|S3\.Bucket|WAF\.Regional\.WebACL|WAF\.WebACL)$"
 )
 LOG_TYPE_REGEX = Regex(
-    r"^(Apache\.AccessCombined|Apache\.AccessCommon|AWS\.ALB|AWS\.AuroraMySQLAudit|AWS"
-    r"\.CloudTrail|AWS\.CloudTrailDigest|AWS\.CloudTrailInsight|AWS\.CloudWatchEvents|AWS"
-    r"\.GuardDuty|AWS\.S3ServerAccess|AWS\.VPCDns|AWS\.VPCFlow|AWS\.WAFWebACL|Box\.Event"
+    r"^(Apache\.AccessCombined|Apache\.AccessCommon|Atlassian\.Audit|AWS\.ALB"
```

Comment:
should the `(` be unterminated?

Reply:
It actually terminates on like 57. This is a wild regex.

Reply:
It actually terminates on line 57.